Userland/BuggieBox/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+serenity_component(
+    BuggieBox
+    REQUIRED
```

Comment:
Is this component actually required?

Reply:
Since I intend this to be compiled and being used in the upcoming initramfs patch, yes, this is quite required :)